compiler/x/env/OMRCPU.hpp

```diff
@@ -55,6 +55,9 @@ class OMR_EXTENSIBLE CPU : public OMR::CPU
    CPU() : OMR::CPU() {}
    CPU(const OMRProcessorDesc& processorDescription) : OMR::CPU(processorDescription) {}
 
+private:
```

Comment:
Consuming projects may need the compilation object at some point.  I'd say make this `protected` or add a getter/setter.